integration/src/main.rs

```diff
@@ -217,7 +217,16 @@ fn run_codegen_test(
         let local_run = if let Some(run) = &suite.run {
             // Reset DB
             reset_db(client)?;
-            cornucopia::load_schema(client, vec![format!("../{}/migrations", run.path)])?;
+            let schema_path = format!("../{}/migrations", run.path);
+            let paths = if std::fs::metadata(&schema_path).unwrap().is_dir() {
+                std::fs::read_dir(&schema_path)
+                    .unwrap()
+                    .map(|it| it.unwrap().path().to_str().unwrap().to_string())
+                    .collect()
```

Comment:
Should be sorted because the order is unstable (i.e. on my machine the migrations are ran in the wrong order causing crash).

Reply:
Out of curiosity, what is your file system?  We should thank your machine for helping so much to discover order problems!

Reply:
I'm running Fedora 36, whose underlying filesystem is BTRFS (I think?). It is mildly exotic, which might be why we're getting such different results :stuck_out_tongue: